docs/source/katalogus/boefjes.rst

```diff
@@ -0,0 +1,64 @@
+=========
+KATalogus
+=========
+The KATalogus contains all boefjes and normalizers that are used in OpenKAT. These can be enabled and disabled. Some boefjes or normalizers require more explanation how they can be used.
+
+Boefjes
+=======
+
+Export to HTTP API
+------------------
+The Export to HTTP API boefje allows the user to export all chosen objects to be sent to an external API endpoint (e.g. webhook). This boefje requires extra settings to be configured.
+
+The clearance level for this boefje indicates that only those objects that are equivalent to the set clearance level, or higher, are shared with the API endpoint.
+
+The clearance level of this boefje only shares those objects that were chosen that have the clearance level of this boefje or higher.
+
+*Example:* if the boefje has clearance level 3 and your chosen objects (e.g. Hostnames) are level 2, these will not be sent to the API end-point. If the boefje clearance level is lowered to level 2, the chosen objects are forwarded to the API endpoint. Giving this boefje a clearance level of 0 will sent all chosen objects to the API endpoint, this includes the objects that have clearance level 0.
+
+Steps to use this boefje:
+
+#. Go to the **KATalogus** and open the **'Export to HTTP API' boefje**.
+
+#. Scroll down to the Container image and **Variants section**.
+
+#. Click on the button **'Add variant'**.
+
+#. **Fill out the form** for creating a Boefje variant. You can find more information :ref:`developer-documentation/development-tutorial/creating-boefje.md:boefje-variants`. The sub-items below are the minimum required items to be added/changed in this form.
```

Comment:
This may fix the build error due to invalid external reference

```suggestion
#. **Fill out the form** for creating a Boefje variant. You can find more information :ref:`developer-documentation/development-tutorial/creating-boefje:Boefje variants`. The sub-items below are the minimum required items to be added/changed in this form.
```